### PR TITLE
Details page

### PR DESCRIPTION
**Pull request by ranasobeid95**

### CODE REVIEW

File: client/src/components/DetailsPage/style.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+} */
+.btns{
+    width: 100vw;
+    height: 13vh;
+    margin-top: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+input{
```
Comment (ahmedisam99) [inline client/src/components/DetailsPage/style.css]: don't select the elements by their tag name

File: client/src/components/DetailsPage/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import imgMeal from '../../assets/shrimppizza.jpg';
+import Button from '../utils/Button';
+import './style.css';
+
+const Details = () => {
```
Comment (ahmedisam99) [inline client/src/components/DetailsPage/index.js]: Use the props coming from the route in order to determine what meal you are trying to show, and send a request to the server to fetch the data associated with that meal

Comment (SajedaIsmail) [inline client/src/components/DetailsPage/index.js]: Make this a class component with state have the meal details in it:
```
state={
meal: {
id: 0,
amount: 0,
price: 0,
name: " ",
vegetables: [ ],
fruits: [ ],
nuts: [ ],
sauce: 0,
salt: 0,
spices: 0,
sugar: 0,
}, }
```
and then update this state depends on the radio buttons, checkboxes, inputs  values that the user chose. Send this meal by the props with the order component when the (show my order button) is clicked.

Have nice coding ^_^

File: client/src/components/App/index.js [hunk 3]
```diff
@@ -1,4 +1,11 @@
 import React from 'react';
+import DetailsPage from '../DetailsPage';
 import './style.css';
 
-export default () => <h1>Hello World</h1>;
+export default () => {
+  return (
+    <div className="pageDetalis">
```
Comment (justinwiley) [inline client/src/components/App/index.js]: It doesn't really matter as long as the class name is the same everywhere, but this should be 'pageDetails'

File: client/src/components/DetailsPage/index.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  return (
+    <div className="details">
+      <header className="header">
+        <nav className="nav">
+          <div className="header__nav">
+            <i className="fas fa-arrow-left"></i>
+            <span className="titel">Make your meal</span>
```
Comment (justinwiley) [inline client/src/components/DetailsPage/index.js]: titel -> title

File: client/src/components/DetailsPage/index.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        Description :
+        <p className="descMeal">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quo eveniet
+          sequi, repudiandae quaerat nobis voluptatem iusto nemo mollitia nam
+          possimus libero impedit magni blanditiis? Esse incidunt voluptatum
+          voluptate quae nisi.
+        </p>
+      </div>
+
+      <div className="mainIngreddients">
```
Comment (justinwiley) [inline client/src/components/DetailsPage/index.js]: mainIngreddients -> mainIngredients

File: client/src/components/DetailsPage/index.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+            </label>
+            <label htmlFor="onions">
+              Onions
+              <input type="checkbox" value="Onions" id="onions" />
+            </label>
+          </div>
+          <div className="spices select">
+            <div>- Spices</div>
+            <label htmlFor="littel">
+              Littel
```
Comment (justinwiley) [inline client/src/components/DetailsPage/index.js]: Littel -> Little.  You could also call this "Small"

File: client/src/components/App/style.css [hunk 7]
```diff
@@ -1,8 +1,45 @@
...
+  font-family: Lato;
+  color: #FFF;
+  flex: 3;
+
+}
+
+.fas{
+  color: #FFF;
+}
+=======
```
Comment (MohammadAlhalaq) [inline client/src/components/App/style.css]:
```suggestion
remove conflict
```

**Pull request merged**